File: (8 kyu) Century From Year/(8 kyu) Century From Year.js
// #1
// const century = (year) => Math.floor(year / 100) + (year % 100 ? 1 : 0);
// #2
// const century = (year) => Math.ceil(year / 100);
// #3
const century = (year) => ((year + 99) / 100) | 0;

File: (8 kyu) Century From Year/(8 kyu) Century From Year.rs
// #1
// fn century(year: u32) -> u32 {
//     year / 100 + if year % 100 == 0 { 0 } else { 1 }
// }
// #2
fn century(year: u32) -> u32 {
    (year - 1) / 100 + 1
}
